cob_bringup/env.sh

```diff
@@ -13,4 +13,6 @@ else
     source /opt/ros/indigo/setup.bash
 fi
 
+export ROS_IP=`hostname -I | awk '{print $1}'`
```

Comment:
@ipa-fxm and @ipa-nhg: do you see any troubles when adding this to our generic `env.sh` script?